### DISCUSSION TITLE
更新包问题

### DISCUSSION THREAD

**Discussion by yemo0399** [note286/xduts#95] [Q&A] [answered]
在更新包时出现了与远端仓库版本不一致，请问下需要更新到2023吗，或者怎么指定更新2022最新版本
![微信图片_20230323174031](https://user-images.githubusercontent.com/128452297/227163385-129bb826-412d-47e2-aa96-62d2cd518a52.png)

**Comment by note286** [comment, marked as answer]
可以选择卸载2022，安装2023。

或者：
```
tlmgr option repository https://mirrors.tuna.tsinghua.edu.cn/tex-historic-archive/systems/texlive/2022/tlnet-final/
tlmgr update --all --self
```

**Reply by yemo0399** [reply]
好的，谢谢大佬！！！！

**Reply by note286** [reply]
给个star呀！(#^.^#)

**Reply by yemo0399** [reply]
必须滴

**Reply by yemo0399** [reply]
！！！！
